### PR TITLE
doc: add minutes for meeting Nov 19 2019

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: meetings/2019-11-19.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+* Michael Dawson (@mhdawson)
+* Joel Chen (@jchip)
+* Bethany Griggs (@BethGriggs)
+* Darcy Clarke (@darcyclarke)
+* Dominykas Blyze (@dominykas)
+* Hassan Sani
+* Wes Todd (@wesleytodd)
+* Edgar Muentes ( @emuentes )
+* Glenn Hinks ( @ghinks )
```
Comment (ljharb) [inline meetings/2019-11-19.md]:
```suggestion
* Glenn Hinks (@ghinks)
* Jordan Harband (@ljharb)
```

File: meetings/2019-11-19.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  * Darcy will try to make sure we are aware of RFC meeting when it is scheduled.
+  * Dominykas: Should we write up documentation on the alternatives to 2FA for publishing staged release versions?
+  * Mentioned: https://blog.travis-ci.com/2019-11-11-build-config-imports
+
+* Future direction support field [#241](https://github.com/nodejs/package-maintenance/issues/241)
+  * Open PR: https://github.com/nodejs/package-maintenance/pull/281
+ * Looks ready to land
+  * Wes, how do we ask people for input
+  * Support just the very basics?
+  * LJ ask a few english questions to figure out which buckets it fits into
```
Comment (ljharb) [inline meetings/2019-11-19.md]:
```suggestion
  * LJ ask a few prose questions to figure out which buckets it fits into
```

File: meetings/2019-11-19.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  * Darcy trying to queue up their product folks to get agreement on what they
+    think they can do.
+  * Within next few weeks will have RFC out. (will be public)
+  * Darcy will try to make sure we are aware of RFC meeting when it is scheduled.
+  * Dominykas: Should we write up documentation on the alternatives to 2FA for publishing staged release versions?
+  * Mentioned: https://blog.travis-ci.com/2019-11-11-build-config-imports
+
+* Future direction support field [#241](https://github.com/nodejs/package-maintenance/issues/241)
+  * Open PR: https://github.com/nodejs/package-maintenance/pull/281
+ * Looks ready to land
```
Comment (dominykas) [inline meetings/2019-11-19.md]:
```suggestion
  * Looks ready to land
```

**Comment by mhdawson** [comment]
#@nodejs/package-maintenance still needs 1 more approval for 7 days waiting period to start.

**Comment by mhdawson** [comment]
Ok 7 days have passed since 4th approver landing.

**Pull request merged**